data_structures/linked_list/add_nodes.py

```diff
@@ -0,0 +1,81 @@
+"""Adding two integers of same length using Linked lists
+
+Example add 321 + 248 = 569
+
+(1)->(2)->(3) + (8)->(4)->(2) = (5)->(6)->(9) """
+
+
+# node definition
+class Node:
+    def __init__(self, data):
```

Comment:
Please add [Python type hints](https://docs.python.org/3/library/typing.html) for function parameters as discussed in [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).